.github/copilot-instructions.md

```diff
@@ -0,0 +1,154 @@
+# Open Domain Specification (ODS)
+
+Open Domain Specification is a Domain-Driven Design (DDD) inspired specification and toolkit for describing business domains in a structured, discoverable, and reusable way. This is a TypeScript monorepo using Lerna for package management and npm workspaces.
+
+Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.
+
+## Working Effectively
+
+### Bootstrap and Build
+- Install dependencies: `npm install` -- takes 5 minutes. NEVER CANCEL. Set timeout to 10+ minutes.
+- Build all packages: `npm run build` -- takes 50 seconds but has known issues (see workarounds below). NEVER CANCEL. Set timeout to 5+ minutes.
+- Run all tests: `npm run test` -- takes 4 seconds. Fast and reliable.
+- Format code: `npx @biomejs/biome check --write` -- takes 40 seconds and flags issues in build files (expected). NEVER CANCEL. Set timeout to 2+ minutes.
+
+### Development Servers
+- Start documentation site: `cd apps/docs && npm run start` -- runs on http://localhost:3000
+- Start ODS UI application: `cd apps/ods-ui && npm run dev` -- runs on http://localhost:5173
+- Start example workspace docs: `cd packages/ods-example-ws && npm run serve` -- runs on random port (displays in output)
+
+### Known Build Issues and Workarounds
+- **ods-example-ws build fails**: Package uses `"type": "module"` but tries to run ts-node which doesn't support ESM properly. 
```

Comment:
This works on my machine what's the error maybe change it from being a module as it just needs to run